mx-rust-semantics/main/representation.md

```diff
@@ -23,17 +25,19 @@ module MX-RUST-REPRESENTATION
                                 | rustValuesToMx(ValueList, MxValueList)
                                 | mxRustLoadPtr(Int)
                                 | mxRustGetBigIntFromStruct(Value)
+                                | mxRustGetStringFromId(Int)
                                 | mxRustNewStruct(MxRustStructType, CallParamsList)
                                   [strict(2), result(ValueWithPtr)]
 
+    syntax TraitType ::= "contract" | "proxy"
     syntax MxRustType ::= "noType" | rustType(Type)
     syntax MxRustTypeOrError ::= MxRustType | SemanticsError
     syntax Value ::= MxRustType
 
-    syntax SemanticsError ::= unknownMxRustType(GenericArg)
-
     syntax Expression ::= concatString(Expression, Expression)  [seqstrict]
                         | toString(Expression)  [strict]
+                        | rawValue(Value)
+                        | SemanticsError  // TODO: Remove.
```

Comment:
Why can't this be removed now? Is this not supposed to be a part of the `mx-semantics` part?

Reply:
I wanted to do this when sending the cleanup PR, but it would require a few hours of work, so I wanted to postpone it until after the deadline.

To be precise, this is related to another TODO in this PR, the one in methods.md at line 280 - those functions use the SemanticsError as an expression. To fix both of them, I would need to make those functions similar to the ones in rust-semantics/conversions.md, returning an `...OrError` result. The problem is that I would also need to change the rule above, in rules.md, at line 267. I would practically need to split the rule in two, the first rule would just rewrite to a new constructor which would also hold the result of `paramsToMaybeTupleElements`, and the second one would actually implement the rule.

If you think it's important to fix this now, I'll send a follow-up PR.